index.html

```diff
@@ -96,10 +101,10 @@ <h2>Vertical Pod Autoscaler</h2>
         <h2 style="text-align:center;">Runs on Any of These Kubernetes Environments</h2>
         <div class="environment">
             <div class="env-box">
-                <img src="assets/aws.png" alt="GitHub">
+                <img src="assets/OpenShift-LogoType.svg.png" alt="Openshift Icon">
```

Comment:
The aws.png image present in the assets folder should be removed if no longer in use.